.changeset/tricky-rivers-return.md

```diff
@@ -0,0 +1,7 @@
+---
+'@chainlink/cfbenchmarks-adapter': minor
+'@chainlink/tiingo-adapter': minor
+'@chainlink/coinmetrics-adapter': minor
```

Comment:
Looks like `coinmetrics` is listed here but missing from the changes, and `ncfx` is missing from the changeset. Can you make sure all 4 have the same aliases and are on the changeset?

Reply:
See [here](https://github.com/smartcontractkit/external-adapters-js/blob/8dfc3ff4509b58924286016adc7703243f9c6df5/packages/sources/coinmetrics/src/endpoint/lwba-ws.ts#L160-L161) for coinmetrics

Reply:
Coinmetrics and ncfx already have the crypto-lwba endpoint name, so I assume no changes needed for those.